### PR TITLE
add in auth options for object storage

### PR DESCRIPTION
**Pull request by rosieyohannan**
IAM role auth options for server 3.3

### CODE REVIEW

File: jekyll/_cci2/server-3-install.adoc [hunk 1]
```diff
@@ -123,32 +123,47 @@ You should have created your S3 Compatible bucket and optional IAM account durin
...
+One of the following is required. Either select IAM keys and provide:
+
 * *Service Account JSON (required)* - 
 A JSON format key of the Service Account to use for bucket access.
 
-* *Storage Object Expiry (optional)* - 
-Number of days to retain your test results and artifacts. Set to 0 to disable and retain objects indefinitely.
+Or select IAM role and provide:
+
+* *Service Account File (required)* - Service account file for the Workload Identity.
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-install.adoc:166]: @circleci/server this line needs checking! Also, is there a good page in the GCP docs we could link to?

Comment (ryanwohara) [reply jekyll/_cci2/server-3-install.adoc:166]: This seems like a good doc for it https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity

Comment (rosieyohannan) [inline jekyll/_cci2/server-3-install.adoc:166]:
```suggestion
* *Service Account File (required)* - Service account file for the https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity[Workload Identity].
```

File: jekyll/_cci2/server-3-install.adoc [hunk 2]
```diff
@@ -123,32 +123,47 @@ You should have created your S3 Compatible bucket and optional IAM account durin
...
+* *Access Key ID (required)* -
+Access Key ID for S3 bucket access.
+
+* *Secret Key (required)* -
+Secret Key for S3 bucket access.
+
+Or select IAM role and provide:
+
+* *Role ARN* - 
+Role ARN (Amazon Resource Name) for S3 bucket access.
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-install.adoc:147]:
```suggestion
https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_identifiers.html[Role ARN] (Amazon Resource Name) for S3 bucket access.
```

**Pull request merged**